checkov/common/sca/reachability/javascript/javascript_alias_mapping_strategy.py

```diff
@@ -0,0 +1,160 @@
+from __future__ import  annotations
+
+import logging
+import os.path
+from abc import ABC
+from typing import List, Dict, Set, Any, Callable
+import re
+import json
+import os
+
+MODULE_EXPORTS_PATTERN = r'module\.exports\s*=\s*({.*?});'
+EXPORT_DEFAULT_PATTERN = r'export\s*default\s*({.*?});'
+
+
+class JavascriptAliasMappingStrategy(ABC):
+
+    @staticmethod
+    def __parse_export(file_content: str, pattern: str) -> Dict[str, Any] | None:
+        module_export_match = re.search(pattern, file_content, re.DOTALL)
+
+        if module_export_match:
+            module_exports_str = module_export_match.group(1)
+            # for having for all the keys and values doube quotes and removing spaces
+            module_exports_str = re.sub(r'\s+', '', re.sub(r'([{\s,])(\w+):', r'\1"\2":', module_exports_str)
+                                        .replace("'", "\""))
+            print(module_exports_str)
```

Comment:
I guess it's for development purposes (?)

Reply:
yep thanks! will be removed